web/src/installers/index.ts

```diff
@@ -51,6 +52,7 @@ export const kubernetesConfigSchema = {
     masterAddress: { type: "string", flag: "kuberenetes-master-address", description: "The address of the internal Kubernetes API server, used during join scripts (read-only)" },
     clusterName: { type: "string", flag: "kubernetes-cluster-name", description: "The name of the Kubernetes cluster (default \"kubernetes\")"},
     loadBalancerAddress: { type: "string", flag: "load-balancer-address", description: "Used for High Availability installs, indicates the address of the external load balancer" },
+    loadBalancerUseFirstPrimary: { type: "boolean", flag: "kubernetes-load-balancer-use-first-primary" , description: "Use first primary address as control plane endpoint in HA cluster" },
```

Comment:
can we put "do not use" in here somewhere?

Reply:
or "for testing upgrades only, use ekco's internal LB instead"

Reply:
done